samples/snippets/linear_regression_tutorial_test.py

```diff
@@ -37,6 +37,21 @@ def test_linear_regression(random_model_id: str) -> None:
         replace=True,
     )
     # [END bigquery_dataframes_bqml_linear_regression]
+    # [START bigquery_dataframes_bqml_linear_evaluate]
+    import bigframes.pandas as bpd
+
+    # Select model you'll use for evaluating. `read_gbq_model` loads model data from
```

Comment:
Probably avoid contractions so you'll -> you will.
Also, maybe "you will be evaluating"